runtime/src/lib.rs

```diff
@@ -585,7 +587,7 @@ impl pallet_transaction_payment::Config for Runtime {
 	type WeightToFee = WeightToCtcFee;
 	type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Self>;
 	type OperationalFeeMultiplier = ConstU8<1u8>;
-	type LengthToFee = LengthToCtcFee;
+	type LengthToFee = ConstantMultiplier<u128, ConstU128<1_500_000_000u128>>;
```

Comment:
Doesn't this affect all extrinsics? The intent of the ticket is to only adjust weight for the System.remark call in particular.

Reply:
Oh, I thought we were doing it to all actually. I figure it wouldn't add much cost to most extrinsics, and this way it covers other extrinsics that could be used for similar purposes (like we'd definitely also have to adjust `System.remark_with_event` at a minimum, and we'd have to check to see if there are others).